fdbserver/TagPartitionedLogSystem.actor.h

```diff
@@ -306,11 +308,10 @@ struct TagPartitionedLogSystem final : ILogSystem, ReferenceCounted<TagPartition
 	LogEpoch getOldestBackupEpoch() const final;
 
 	void setOldestBackupEpoch(LogEpoch epoch) final;
-
 	ACTOR static Future<Void> monitorLog(Reference<AsyncVar<OptionalInterface<TLogInterface>>> logServer,
 	                                     Reference<AsyncVar<bool>> failed);
 
-	Optional<std::pair<Version, Version>> static getDurableVersion(
+	Optional<std::tuple<Version, Version, std::vector<TLogLockResult>>> static getDurableVersion(
```

Comment:
Nit: can you add comment for this function, especially on meaning of the return value?